### DISCUSSION TITLE
Share facial recognition among users

### DISCUSSION THREAD

**Discussion by sowensupgrade** [immich-app/immich#14933] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

User 1 matches all faces to a name. User 2, 3, ... N all see these mappings that the other users make. If you are sharing an Immich server... then you will likely all agree on the name a person goes by when matching to their face.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#12614
